api/src/convert/block_contents.rs

```diff
@@ -18,8 +23,18 @@ impl From<&mc_transaction_core::BlockContents> for blockchain::BlockContents {
         let outputs: Vec<external::TxOut> =
             source.outputs.iter().map(external::TxOut::from).collect();
 
+        let set_mint_config_txs: Vec<_> = source
+            .set_mint_config_txs
+            .iter()
+            .map(external::SetMintConfigTx::from)
+            .collect();
+
+        let mint_txs: Vec<_> = source.mint_txs.iter().map(external::MintTx::from).collect();
+
         block_contents.set_key_images(RepeatedField::from_vec(key_images));
         block_contents.set_outputs(RepeatedField::from_vec(outputs));
+        block_contents.set_set_mint_config_txs(RepeatedField::from_vec(set_mint_config_txs));
```

Comment:
Optional/Equivalent: Omit the `Vec<_>` declaration to get a `RepeatedField`, which implements `FromIterator`

```suggestion
        let outputs = source.outputs.iter().map(external::TxOut::from).collect();
        let set_mint_config_txs = source
            .set_mint_config_txs
            .iter()
            .map(external::SetMintConfigTx::from)
            .collect();

        let mint_txs = source.mint_txs.iter().map(external::MintTx::from).collect();

        block_contents.set_key_images(key_images);
        block_contents.set_outputs(outputs);
        block_contents.set_set_mint_config_txs(set_mint_config_txs);
```

Reply:
Nice